app/templates/components/forms/admin/settings/system-form.hbs

```diff
@@ -0,0 +1,79 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'App Environment'}}
+  </h3>
+  <div class="grouped fields">
+    <label>
+      {{t 'Where is the application running ?'}}
+    </label>
+    <div class="field">
+      {{ui-radio label=(t 'Development') current='development' name='environment' value='development'}}
+    </div>
+    <div class="field">
+      {{ui-radio label=(t 'Staging') name='environment' value='staging'}}
+    </div>
+    <div class="field">
+      {{ui-radio label=(t 'Production') name='environment' value='production'}}
+    </div>
+  </div>
+  <h3 class="ui header">
+    {{t 'App Name'}}
+  </h3>
+  <div class="grouped fields">
```

Comment:
Remove this `<div class="grouped fields">` completely. The fields would still be displayed correctly. Not required for normal input fields. Generally used for checkboxes and radios.